.clang-format

```diff
@@ -0,0 +1,3 @@
+Language: Cpp
+BasedOnStyle: GNU
+ColumnLimit: 99
```

Comment:
Can we add https://github.com/coreos/rpm-ostree/blob/deb8b3b155abcb546712d7f610f68d032d6e1366/.clang-format#L5 here? (See https://github.com/coreos/rpm-ostree/pull/3604.)

Reply:
Sure, I synchronized them exactly to also bump the column limit to 100 just for consistency.